src/CoverageBase.jl

```diff
@@ -5,16 +5,22 @@ export testnames, runtests
 
 const need_inlining = []
 
+if VERSION < v"0.7.0-DEV.3073"
```

Comment:
why not use Compat's binding for this?

Reply:
Seemed unnecessary to add a dependency on Compat for that one thing, but I guess we have an indirect dependency on it from Coverage so I should just do that.